src/main/resources/fabric.mod.json

```diff
@@ -18,21 +18,15 @@
   "license": "All Rights Reserved",
 
   "environment": "*",
-  "entrypoints": {
-    "client": [
-      "example.mod.client.ExampleModClient"
-    ],
-    "main": [
-      "example.mod.ExampleMod"
-    ],
-    "journeymap": [
-      "example.mod.client.plugin.ExampleJourneymapPlugin"
-    ]
-  },
   "depends": {
     "fabricloader": ">=0.11.3",
     "fabric": "*",
     "minecraft": "1.18.x",
     "java": ">=17"
+  },
+  "custom": {
+    "modmenu": {
+      "parent": "journeymap-fabric"
+    }
```

Comment:
What is this for?

Reply:
This makes it so that the JourneyMap API and JourneyMap mods don't appear as separate mods in Mod Menu.